<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewProjectDialog</class>
 <widget class="QDialog" name="NewProjectDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>804</width>
    <height>161</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>New Project</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(60, 63, 65);
color: rgb(230, 230, 230);


</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Name:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Communication:</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLineEdit" name="projectNameLineEdit">
             <property name="styleSheet">
              <string notr="true">QLineEdit {
    border: 2px solid rgb(230, 230, 230);
    border-radius: 1px;
}
QLineEdit:hover {
	background-color: rgb(64, 64, 64);
	border: 2px solid #8f8f91;
	border-color: rgb(60, 187, 255);
}

QLineEdit:focus {
	background-color: rgb(64, 64, 64);
	border: 2px solid #8f8f91;
	border-color: rgb(60, 187, 255);
}</string>
             </property>
             <property name="text">
              <string>untitled</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="communicationTypeComboBox">
             <property name="styleSheet">
              <string notr="true">QComboBox {
    border: 2px solid rgb(230, 230, 230);
    border-radius: 1px;
}
QComboBox:hover {
	background-color: rgb(64, 64, 64);
	border: 2px solid #8f8f91;
	border-color: rgb(60, 187, 255);
}

QComboBox:focus {
	background-color: rgb(64, 64, 64);
	border: 2px solid #8f8f91;
	border-color: rgb(60, 187, 255);
}</string>
             </property>
             <item>
              <property name="text">
               <string>Serial Communication ( COM )</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Bluetooth Comunication</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Ethernet Communication</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Create in:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="projectPathLineEdit">
           <property name="styleSheet">
            <string notr="true">QLineEdit {
    border: 2px solid rgb(230, 230, 230);
    border-radius: 1px;
}
QLineEdit:hover {
	background-color: rgb(64, 64, 64);
	border: 2px solid #8f8f91;
	border-color: rgb(60, 187, 255);
}

QLineEdit:focus {
	background-color: rgb(64, 64, 64);
	border: 2px solid #8f8f91;
	border-color: rgb(60, 187, 255);
}</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="browseButton">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
    border: 2px solid #8f8f91;
    border-radius: 3px;
	height: 30px;
	width: 60;
}
QPushButton:hover {
	background-color: rgb(60, 187, 255);
}</string>
           </property>
           <property name="text">
            <string>Browse</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="createButton">
         <property name="styleSheet">
          <string notr="true">QPushButton {
    border: 2px solid #8f8f91;
    border-radius: 3px;
	height: 30px;
	width: 60;
}
QPushButton:hover {
	background-color: rgb(60, 187, 255);
}</string>
         </property>
         <property name="text">
          <string>Create</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="cancelButton">
         <property name="styleSheet">
          <string notr="true">QPushButton {
    border: 2px solid #8f8f91;
    border-radius: 3px;
	height: 30px;
	width: 60;
}
QPushButton:hover {
	background-color: rgb(60, 187, 255);
}
</string>
         </property>
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>NewProjectDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>644</x>
     <y>99</y>
    </hint>
    <hint type="destinationlabel">
     <x>343</x>
     <y>63</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
